hack/make-rules/test-cmd-util.sh

```diff
@@ -3920,7 +3921,8 @@ run_pod_templates_tests() {
   # Command
   kubectl delete podtemplate nginx "${kube_flags[@]}"
   # Post-condition: No templates exist
-  kube::test::get_object_assert podtemplate "{{range.items}}{{.metadata.name}}:{{end}}" ''
+  # TODO nilebox: looks like even 10 tries is sometimes not enough (GC is too busy?)
+  kube::test::wait_long_object_assert podtemplate "{{range.items}}{{.metadata.name}}:{{end}}" ''
```

Comment:
Does the podtemplate have any dependents? If not, GC should have handled the deletion quickly.

Reply:
@caesarxuchao doesn't look like it (`podtemplate.json`):
```json
  {
    "apiVersion": "v1",
    "kind": "PodTemplate",
    "metadata": {
      "name": "nginx"
    },
    "template": {
      "metadata": {
        "labels": {
          "name": "nginx"
        },
        "generateName": "nginx-"
      },
      "spec": {
         "containers": [{
           "name": "nginx",
           "image": "dockerfile/nginx",
           "ports": [{"containerPort": 80}]
         }]
      }
    }
  }
```
But the assert still fails for some reason.